docs/esp32/quickref.rst

```diff
@@ -58,12 +61,19 @@ The :mod:`esp32` module::
     import esp32
 
     esp32.raw_temperature() # read the internal temperature of the MCU, in Fahrenheit
-    esp32.ULP()             # access to the Ultra-Low-Power Co-processor
+    esp32.ULP()             # access to the Ultra-Low-Power Co-processor, not on ESP32C3
 
 Note that the temperature sensor in the ESP32 will typically read higher than
 ambient due to the IC getting warm while it runs.  This effect can be minimised
 by reading the temperature sensor immediately after waking up from sleep.
 
+ESP32C3, ESP32S2, and ESP32S3 also have an internal sensor available.
+It is implemented a bit differently than the ESP32 and returns
+temperature in Celsius::
+
+    esp32.mcu_temperature() # read the internal temperature of the MCU, in Celsius
```

Comment:
Called `mcu_temperature` because initial discussion had some thinking of using it for monitoring ambient temp- not core temp.  (tcore - offset?)   Not particularly attached to that function name though. The older ESP32 version (older IDF library) uses `raw_temperature`.

Reply:
I see, I just had a look at the discussion. So the idea is that someone might incorrectly assume `esp32.temperature()` is measuring the ambient temperature? I guess I can see how someone could reach that conclusion, OK.

I agree that making a new function name is a good idea. `raw_temperature()` is janky, unsupported by Espressif, and returns Fahrenheit (or Fahrenheit-ish really). :laughing: